src/gta/net_game_event.hpp

```diff
@@ -530,6 +530,7 @@ namespace rage
 		alignas(8) bool m_is_headshot; //0x0050
 		alignas(8) bool m_is_with_melee_weapon; //0x0058
 		alignas(8) int m_hit_material; //0x0060
+		char pad_[0x43]; // !FIXME: THIS IS AWFUL BUT I'M TOO STUPID TO FIX IT. tupoy.
```

Comment:
This has to be fixed.

Reply:
`static_assert(sizeof(sEntityDamagedData) == 0x68);` will fail otherwise because size will be 0x25

Reply:
I know that padding doesn't fix it, i just added padding so it can compile.